src/main/thrift/parquet.thrift

```diff
@@ -197,18 +197,78 @@ enum FieldRepetitionType {
   REPEATED = 2;
 }
 
+/** Identifier for the sort order used to produce min and max values. */
+enum Order {
+  /**
+   * The signed ordering is the order produced by comparing single primitive
+   * values with a signed comparator, or the lexicographic ordering produced by
+   * comparing each byte of a binary or fixed using a signed comparator.
+   *
+   * (A signed comparator uses the most-significant bit as a sign bit; an
+   * unsigned comparator uses the most-significant bit as part of the value's
+   * magnitude. Note that unsigned comparison is not defined for floating
+   * point values.)
+   */
+  SIGNED = 0;
+
+  /**
+   * The unsigned ordering is produced by comparing single primitive values
+   * with an unsigned comparison, or the lexicographic ordering produced by
+   * comparing eacy byte of a binary or fixed using an unsigned comparator.
+   *
+   * (A signed comparator uses the most-significant bit as a sign bit; an
+   * unsigned comparator uses the most-significant bit as part of the value's
+   * magnitude. Note that unsigned comparison is not defined for floating
+   * point values.)
+   */
+  UNSIGNED = 1;
+
+  /**
+   * Identifiers for custom orderings, to be defined using CustomOrder entries
+   * in the file footer.
+   */
+  CUSTOM1 = 2;
+  CUSTOM2 = 3;
+  CUSTOM3 = 4;
+  CUSTOM4 = 5;
+  CUSTOM5 = 6;
+  CUSTOM6 = 7;
```

Comment:
I don't think the sort order should be allowed to change per page. So it should not be in the page metadata but in the footer per column. For example in ColumnMetaData or in SchemaElement.
I would tend to pick SchemaElement.
As a separate note, if it is an id to lookup in a table then we should make it an integer instead of an enum.